### PR TITLE
KDUMP test port

### PR DESCRIPTION
**Pull request by mbivolan**

### CODE REVIEW

File: Testscripts/Linux/KDUMP-Config.sh [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+            fi
+        ;;
+        suse*)
+            zypper refresh; zypper --non-interactive install kexec-tools kdump makedumpfile
+            if [ $? -ne 0 ]; then
+                UpdateSummary "Warning: Kexec-tools failed to install."
+            fi
+        ;;
+        *)
+            msg=
```
Comment (chvalean) [inline Testscripts/Linux/KDUMP-Config.sh]: to be removed

File: Testscripts/Linux/KDUMP-Config.sh [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+    value_kdump=(
+        ''
+        '1'
+        ''
+        '0'
+    )
+
+    kdump_commandline_arguments=$(grep KDUMP_COMMANDLINE_APPEND $kdump_sysconfig |  sed 's/KDUMP_COMMANDLINE_APPEND="//g' | sed 's/"//g')
+
+
```
Comment (adriansuhov) [inline Testscripts/Linux/KDUMP-Config.sh]: 2 blank lines

File: Testscripts/Linux/KDUMP-Config.sh [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+        ;;
+    esac
+}
+#######################################################################
+#
+# RhelExtraSettings()
+#
+#######################################################################
+RhelExtraSettings(){
+
```
Comment (adriansuhov) [inline Testscripts/Linux/KDUMP-Config.sh]: Try to remove blank lines where is possible or doesn't break code readability

File: Testscripts/Linux/KDUMP-Config.sh [hunk 4]
```diff
@@ -0,0 +1,401 @@
...
+        ;;
+        suse*)
+            zypper refresh; zypper --non-interactive install kexec-tools kdump makedumpfile
+            if [ $? -ne 0 ]; then
+                UpdateSummary "Warning: Kexec-tools failed to install."
+            fi
+        ;;
+        *)
+            msg=
+            LogErr "Warning: Distro '${distro}' not supported. Kexec-tools failed to install."
```
Comment (asofron) [inline Testscripts/Linux/KDUMP-Config.sh]: Replace with LogWarn

File: Testscripts/Linux/KDUMP-Config.sh [hunk 5]
```diff
@@ -0,0 +1,401 @@
...
+
+    sed -i '/^path/ s/path/#path/g' $kdump_conf
+    if [ $? -ne 0 ]; then
+        LogErr "Failed to comment path in /etc/kdump.conf. Probably kdump is not installed."
+        SetTestStateAborted
+        exit 0
+    else
+        echo path $dump_path >> $kdump_conf
+        LogMsg "Success: Updated the path to /var/crash."
+        UpdateSummary "Success: Updated the path to /var/crash."
```
Comment (asofron) [inline Testscripts/Linux/KDUMP-Config.sh]: UpdateSummary should be removed

File: Testscripts/Linux/KDUMP-Config.sh (hunk 4 above)
Comment (adriansuhov) [reply Testscripts/Linux/KDUMP-Config.sh]: LogWarn is not available for bash scripts, only for ps1 afaik

**Pull request merged**